spec/integration/dry/configurable/setting_spec.rb

```diff
@@ -328,7 +328,7 @@
         klass.setting :env
 
         klass.setting :db do
-          setting :user, "root"
+          setting :user, +"root"
```

Comment:
what...does this do?

Reply:
/shrug [unfreezing the string](https://ruby-doc.org/core-2.7.2/String.html#method-i-2B)
Changed to `dup` to make it less confusing.